packages/PlanLimitsUI/src/pages/Limits/locationString.spec.ts

```diff
@@ -69,12 +63,6 @@ describe('createLocationString', () => {
 });
 
 describe('parsePinnedLocation', () => {
-  it('should return null when null passed', () => {
-    const result = parsePinnedLocation(null);
-
-    expect(result).toEqual(null);
-  });
-
```

Comment:
TSC tells me these are redundant after tightening up the typing of the functions being tested.